app/scripts/migrations/104.ts

```diff
@@ -0,0 +1,48 @@
+import { hasProperty, isObject } from '@metamask/utils';
+import { captureException } from '@sentry/browser';
+import { cloneDeep } from 'lodash';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 104;
+
+/**
+ * Remove network controller `networkId` state.
```

Comment:
I think this description needs to be updated

Reply:
@danjm I updated the description. Thanks 😄